docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po

```diff
@@ -0,0 +1,207 @@
+msgid ""
+"In order to `Create a new recovery device`_ or `Use a recovery device`_ you "
+"need to open the recovery device dialog from the main menu:"
+msgstr ""
+"Pour pouvoir `Créer un nouvel appareil de récupération`_ ou d'`Utiliser un "
+"appareil de récupération`_ vous avez besoin d'ouvrir le dialogue de "
+"récupération d'appareil dans le menu principal:"
+
+#: ../../userguide/recovery_device.rst:13
+msgid "Open the ``Recovery device`` menu."
+msgstr "Ouvrir le menu ``Récupération d'appareils``."
+
+#: ../../userguide/recovery_device.rst:15
+msgid ""
+"You need to open the sub-menu then click on ``Recovery device`` option "
+"(alternatively, you can use the shortcut ``Ctrl+I``)."
+msgstr ""
+"Vous avez besoin d'ouvrir le sous-menu puis cliquer sur ``récupération "
+"d'appareil`` (Alternativement, vous pouvez utiliser le raccourci ``Ctrl+I``)."
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Parsec menu with recovery device option highlighted"
+msgstr "Le menu Parsec avec l'option récupération d'appareil surligné"
+
+#: ../../userguide/recovery_device.rst:20
+msgid "That will open a modal dialog that looks like this:"
+msgstr "Ça va ouvrir un dialogue qui ressemble à ceci:"
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Parsec recovery device modal"
+msgstr "Dialogue Parsec de récupération d'appareil"
+
+#: ../../userguide/recovery_device.rst:26
+msgid "Create a new recovery device"
+msgstr "Créer un nouvel appareil de récupération"
+
+#: ../../userguide/recovery_device.rst:28
+msgid "In the Device recovery dialog, select ``Create a recovery device``"
+msgstr ""
+"Dans le dialogue de récupération d'appareil, sélectionner ``Créer un "
+"appareil de récupération``"
+
+#: ../../userguide/recovery_device.rst:30
+#: ../../userguide/recovery_device.rst:61
+msgid "From here:"
+msgstr "À partir de là:"
+
+#: ../../userguide/recovery_device.rst:32
+msgid "Select the device you would like to create a recovery device for."
+msgstr ""
+"Sélectionner l'appareil à partir duquel vous voulez créer un appareil de "
+"récupération."
+
+#: ../../userguide/recovery_device.rst:33
+msgid ""
+"Click the ``Select`` button to specify a file to store the recovery device."
+msgstr ""
+"Cliquer sur le bouton ``Choisir`` pour spécifier le fichier où sera "
+"sauvegardé l'appareil de récupération."
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Create a recovery device modal"
+msgstr "Dialogue de création d'un appareil de récupération"
+
+#: ../../userguide/recovery_device.rst:38
+msgid ""
+"Click ``Next``, you will be prompted to enter the password for the device "
+"you have selected"
+msgstr ""
+"Cliquer sur ``Suivant``, vous serez demandé d'entrer le mot de passe pour "
+"l'appareil que vous avez sélectionné"
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Create a recovery device re-authenticate modal"
+msgstr ""
+"Dialogue de reconnection pour la création d'un appareil de récupération"
```

Comment:
```suggestion
"Seconde authentification pour la création d'un appareil de récupération"
```

Reply:
When creating a `recovery device`, you only have a single authentication dialog and that's this one.

Reply:
Translation ("reconnection") differ from English msg ("re-authenticate").

Reply:
outdated